src/views/AdminEventLog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+    <div>
+      <v-text-field
+          v-model="search"
+          append-icon="mdi-magnify"
+          label=" Search"
+      ></v-text-field>
+        <v-data-table
+                :headers="headers"
+                :items="adminEvents"
+                :items-per-page="15"
+                class="elevation-1"
+                show-expand
+                :single-expand="singleExpand"
+                item-key="key"
+                loading-text="Loading events"
+                :loading="loadingStatus"
+                :search="search"
+        >
+            <template v-slot:expanded-item="{ headers, item }">
+                <td :colspan="headers.length"><pre>{{item.representation | pretty}}</pre></td>
+            </template>
+        </v-data-table>
+        <button @click="getAdminEvents">Refresh</button>
+    </div>
+</template>
+
+<script>
+    import AdminEventsRepository from "../api/AdminEventsRepository";
```

Comment:
I'm ok to pull out the factory and just import. 
Lets make it consistent across the app.